src/contexts/AuthContext.tsx

```
@@ -0,0 +1,158 @@
+  const appDispatch = useAppDispatch();
+  const [state, dispatch] = useReducer(reducer, initialState);
+
+  const initialize = async () => {
+    try {
+      const response = await appDispatch(getProfile()).unwrap();
+      const userInfo = response.data;
+      dispatch({
+        type: "INITIALIZE",
+        payload: { isAuthenticated: true, user: userInfo },
+      });
+    } catch (e: any) {
```

ESLint

[error] 'e' is defined but never used.

(@typescript-eslint/no-unused-vars)